meterfs/meterfs.c

```diff
@@ -1154,6 +1167,34 @@ int meterfs_resizeFile(const char *name, size_t filesz, size_t recordsz, meterfs
 }
 
 
+int meterfs_resetFile(file_t *f, meterfs_ctx_t *ctx)
+{
+	fileheader_t hdr;
+
+	if (meterfs_getFileInfoName(f->header.name, &hdr, ctx) < 0) {
+		return -ENOENT;
+	}
+
+	if (!hdr.sector || !hdr.sectorcnt) {
+		return -EFAULT;
+	}
+
+	hdr.uid = hdr.uid + 1;
+
+	if (hdr.uid == 0xffffffff) {
+		/* Can't allow file to have uid it had before */
+		return -EPERM;
+	}
+
+	/* Set all current entries to be ignored by read ops */
+	hdr.firstid = f->lastidx.no + 1;
+
+	meterfs_powerCtrl(1, ctx);
```

Comment:
I kinda don't like that we change the global state permanently.

On error we don't really know if power has been enabled (`meterfs_updateOneFile` might've returned error, this is after enable)